contracts/test/Multisig.test.ts

```diff
@@ -23,8 +23,6 @@ describe("Multisig with single owner", function () {
   let privateKey: string;
   let publicKey: string;
 
-  let txIndex = -1; // faster to track this internally than to request from contract
```

Comment:
Why was this dropped? Seems like it would be a good thing for test extendability to increment this instead of using hardcoded values for txIndex.

Reply:
Ah but we're not using hardcoded values - we're retrieving it from the blockchain. This construct was there earlier because all blockchain interaction was very slow, so it was faster to track this value in our code. But now blockchain interaction is a bit faster, so it's safer to remove this kind of dependencies. The tests are more idempotent without this.